dashboard/memory_utils.py

```diff
@@ -391,10 +427,14 @@ def memory_summary(state,
         mem += f"--- Object references for {group_by}: {key} ---\n"
         mem += object_ref_string\
             .format(*object_ref_labels)
+        n = 0
         for entry in group["entries"]:
+            n += 1
```

Comment:
Can you comment what this line is for?

Reply:
Gotcha. I'll add that `n` is a counter for num entries per group.